################################################################################
#          This file was automatically generated. Please do not edit.          #
################################################################################

import jpype



IndexedDbaseFileReader = jpype.JClass('org.geotools.data.shapefile.dbf.IndexedDbaseFileReader')

DbaseFileWriter = jpype.JClass('org.geotools.data.shapefile.dbf.DbaseFileWriter')

DbaseFileReader = jpype.JClass('org.geotools.data.shapefile.dbf.DbaseFileReader')

DbaseFileHeader = jpype.JClass('org.geotools.data.shapefile.dbf.DbaseFileHeader')

DbaseFileException = jpype.JClass('org.geotools.data.shapefile.dbf.DbaseFileException')